.github/workflows/ort.yml

```diff
@@ -0,0 +1,21 @@
+name: ORT
+
+on:
+  workflow_dispatch:
+
+env:
+  ORT_IMAGE: ghcr.io/oss-review-toolkit/ort-minimal
+
+jobs:
+  ort:
+    name: Run ORT
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Pull ORT Docker Image
+      run: docker pull ${{ env.ORT_IMAGE }}
```

Comment:
Is it really necessary to download the massive Docker image? Can't we just download the distribution archive instead, and install eventually missing tools via setup actions?

Reply:
Maybe I'll give it a try, for simplicity I wanted to start with the Docker image.

Reply:
I takes about a minute to pull the minimal image, I'll compare that to the other approach which should be faster.